web/src/components/organisms/Settings/Members/index.tsx

```diff
@@ -154,10 +157,13 @@ const Members: React.FC = () => {
       <MemberAddModal
         open={MemberAddModalShown}
         searchedUser={searchedUser}
+        searchedUserList={searchedUserList}
+        changeSearchedUserList={changeSearchedUserList}
         onClose={handleMemberAddModalClose}
         handleUserSearch={handleUserSearch}
+        handleUserAdd={handleUserAdd}
```

Comment:
`onUserAdd`
Also, please update the prop above it too, handleUserSearch => onUserSearch

Reply:
Done!